From 6c2c841aa260837bbe2d2aecde4968d90ea726d6 Mon Sep 17 00:00:00 2001
From: Jean-Christophe Morin <jcmorin@anaconda.com>
Date: Mon, 4 Dec 2023 18:07:28 -0500
Subject: [PATCH] Fix encoding error on Windows

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 2660a66..24a71aa 100755
--- a/setup.py
+++ b/setup.py
@@ -23,7 +23,7 @@ reqs = get_reqs("requirements.txt")
 cli_reqs = get_reqs("requirements-cli.txt")
 optimize_reqs = get_reqs("requirements-optimize.txt")
 
-with open('README.md') as file:
+with open('README.md', encoding='utf-8', errors='replace') as file:
     long_description = file.read()
 
 
-- 
2.43.0

